src/main/java/ch/njol/skript/expressions/ExprBookPages.java

```diff
@@ -54,38 +54,58 @@
 		Skript.registerExpression(ExprBookPages.class, String.class, ExpressionType.PROPERTY,
```

Comment:
Support for getting multiple specific pages?
```
[book] page[s] %numbers% of %itemtypes%
```
Also, do you think this would be useful?
```
[the] %number%(st|nd|rd|th) page of %itemtypes%
%itemtypes%'[s] %number%(st|nd|rd|th) page
```

Reply:
> Support for getting multiple specific pages?
> 
> ```
> [book] page[s] %numbers% of %itemtypes%
> ```

I was initially doing this but stumbled upon a fundamental issue I can't seem to remember right now, guess I gotta do it again and see if I remember.

> Also, do you think this would be useful?
> 
> ```
> [the] %number%(st|nd|rd|th) page of %itemtypes%
> %itemtypes%'[s] %number%(st|nd|rd|th) page
> ```

This looks extremely ugly with variables `{_n}th page of {_some-book}` and if we make it optional, people are not going to use it so, it's whatever.